gerrymandering/gerrymandering.ipynb

```diff
@@ -169,18 +176,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "opts.defaults(\n",
-    "    opts.Polygons(fill_alpha=0))"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "shaded = datashade(census_points, **shade_defaults)\n",
-    "tiles * shaded * color_points * districts"
+    "districts * shaded * color_points"
```

Comment:
Can you verify this output (i.e. paste screenshot here). I don't see shaded using `geo=True`, and tiles would project to `ccrs.GOOGLE_MERCATOR`